frontend/src/components/SimpleRange/SimpleRange.js

```diff
@@ -0,0 +1,48 @@
+import { FormattedNumber } from 'react-intl'
+import { styled } from '@mui/material/styles'
+import LinearProgress, { linearProgressClasses } from '@mui/material/LinearProgress'
+import Tooltip from '@mui/material/Tooltip'
+import PropTypes from 'prop-types'
+
+const GradientProgress = styled(LinearProgress)(({ theme, value }) => ({
+  [`& .${linearProgressClasses.bar}`]: {
+    overflow: 'hidden',
+    backgroundColor: 'transparent',
+    '&:before': {
+      content: '""',
+      display: 'block',
+      width: '100%',
+      height: '100%',
+      transition: 'transform .4s linear',
+      transform: `translateX(${100 - value}%)`,
+      background: `linear-gradient(270deg, ${theme.palette.canary.main} 0%, ${theme.palette.chestnutRose.main} 50%, ${theme.palette.ultramarine.main} 100%)`, // eslint-disable-line
+    },
+  },
+}))
+
+const SimpleRange = ({ min, max, value, style, color }) => {
+  const Range = style === 'gradient' ? GradientProgress : LinearProgress
```

Comment:
Both the outer and inner bars of SimpleRanges should have rounded corners. The inner bar is missing them in all browsers and there's a rendering difference in Safari which makes the outer bar not have rounded corners.

You can fix all of that if you add `borderRadius: '4px'` to LinearProgress-bar, `borderRadius: '4px'` to GradientProgress's `'&:before'`, and use GradientProgress for both gradient and solid style SimpleRanges.

That last bit is required to make the left-side corners rounded on Safari. Something about putting the fill in a `'&:before'` pseudo-element makes Safari round its corners properly.